modules/dedicated-creating-your-cluster.adoc

```diff
@@ -39,4 +39,35 @@ To access your {product-title} cluster:
 . From link:https://cloud.redhat.com/openshift[cloud.redhat.com/openshift], click
  on the cluster you want to access.
 
- . Click *Launch Console*.
+ . Click *Open Console*.
+
+= Configuring your identity providers
+
+To configure identity providers for your {product-title} cluster:
+
+. Navigate to the *Cluster* page and *Access control* tab.
+
+. Click the *Add identity provider* button.
+
+. Select the identity provider from the drop-down menu.
+
+. Enter a unique name for the identity provider. This name cannot be changed later.
+
+. Select a mapping method from the drop-down menu.
+
+. Enter a client ID and client secret.
+
+. (Optional) Enter a hostname to connect to either GitHub or GitHub Enterprise.
```

Comment:
s/. (Optional) Enter/. Optional: Enter/

IIRC, we opted to format optional steps or annotation definitions using a colon:
`. Optional: Enter a hostname...`